examples/pizzas/configuration/pizza-orders-configmap.yaml

```
@@ -111,3 +111,28 @@ data:
         "filter": "$root.spec.pizzas[name = $item.metadata.name and namespace = $item.metadata.namespace]"
       }
     }
+  presets: |-
+    [
+      {
+        "name": "cash-delivery",
+        "value": {
+          "spec": {
+            "orderDetails": {
+              "paymentMethod": "CASH",
+              "realization": "DELIVERY",
+            }
+          }
+        }
+      },
+      {
+        "name": "cars-pickup",
```

```suggestion
        "name": "card-pickup",
```
I know, I'm nitpicking, but let's keep our examples neat.